src/hardware/mixer.cpp

```diff
@@ -2157,6 +2157,20 @@ class MIXER final : public Program {
 		        "Examples:\n"
 		        "  [color=green]mixer[reset] [color=cyan]cdda[reset] [color=white]50[reset] [color=cyan]sb[reset] [color=white]reverse[reset] /noshow\n"
 		        "  [color=green]mixer[reset] [color=white]x30[reset] [color=cyan]fm[reset] [color=white]150 r50 c30[reset] [color=cyan]sb[reset] [color=white]x10[reset]");
+
+		MSG_Add("SHELL_CMD_MIXER_HEADER_DELINEATION",
+		        "%-22s %4.0f:%-4.0f %+6.2f:%-+6.2f  %-8s %5s %7s %7s\n");
```

Comment:
I don't think the trailing newlines should be part of single-line messages because then we're depending on people not leaving them off. Of course, multi-line messages are exceptions.

Reply:
Moved the delimiter field back into the code. I figured the spacing is driven by the numbers (primarily) and not by the English column names. So I'd rather translators find words (or abbreviations, if needed) to fit - and keep the numbers formatted per your original layout.

(If a particular language becomes a problem, we can figure it out then.. but hopefully won't be needed).